bin/assess_mash_screen.py

```diff
@@ -1,207 +0,0 @@
-#!/usr/bin/env python3
-"""
-Assess MASH screening results.
-"""
-__author__ = "Fredrik Boulund"
-__date__ = "2018"
-__version__ = "0.6.1b"
-
-from sys import argv, exit, stdout, stderr
```

Comment:
@boulund @emilio-r , since this file isn't being used anymore - I have removed this. 

Please let me know if you'd like me to keep it.